lib/class-apc-cache-interceptor.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+// phpcs:disable Universal.Files.SeparateFunctionsFromOO.Mixed
```

Comment:
Not sure if this file is copied in from elsewhere, but separating it into a class-only file and a functions file may be the better approach here IMO.